ui/src/app/pages/login/account-creation/account-creation.tsx

```diff
@@ -439,7 +439,7 @@ export class AccountCreation extends React.Component<AccountCreationProps, Accou
                          onChange={v => this.updateProfile('givenName', v)}/>
               {givenName.length > nameLength &&
               <ErrorMessage id='givenNameError'>
-                First Name must be 80 characters or less.
+                First Name must be {nameLength} characters or less.
```

Comment:
nit: this should be fewer rather than less since it's countable in a reasonable period of time.